### DISCUSSION TITLE
Explanation of docker image options?

### DISCUSSION THREAD

**Discussion by ireuben** [blakeblackshear/frigate#7847] [General]
OK, this might be a silly question, but I can't find anywhere that explains what the different Docker images are for:

ghcr.io/blakeblackshear/frigate:0.13.0-beta1
ghcr.io/blakeblackshear/frigate:0.13.0-beta1-standard-arm64
ghcr.io/blakeblackshear/frigate:0.13.0-beta1-tensorrt
ghcr.io/blakeblackshear/frigate:0.13.0-beta1-tensorrt-jp4
ghcr.io/blakeblackshear/frigate:0.13.0-beta1-tensorrt-jp5

I'm pretty sure the tensorrt ones don't apply to me (google says it's an NVIDIA thing), but I do have an M1 Mac and so have a arm64 architecture. But I've always been using the most basic Docker image. Should I be using "standard-arm64"? What does the "standard" part of that mean?

**Comment by NickM-27** [comment]
This is covered in the docs https://deploy-preview-6262--frigate-docs.netlify.app/frigate/installation#docker

The vast majority of users should just use the default image
